File: untold_comparison_programs/06_http_requests/python/main.py
# HTTP Requests
import requests

# GET request
print("Fetching data...")
res = requests.get("https://api.github.com")
print(f"Status: {res.status_code}")
print(f"Body length: {len(res.text)}")

# POST request
post_res = requests.post("https://httpbin.org/post", json={"key": "value"})
print(f"POST Status: {post_res.status_code}")

File: untold_comparison_programs/06_http_requests/javascript/main.js
// HTTP Requests
const axios = require('axios');

async function main() {
    // GET request
    console.log("Fetching data...");
    const res = await axios.get("https://api.github.com");
    console.log(`Status: ${res.status}`);
    console.log(`Body length: ${res.data.length}`);

    // POST request
    const postRes = await axios.post("https://httpbin.org/post", {key: "value"});
    console.log(`POST Status: ${postRes.status}`);
}

main();